### PR TITLE
remove the need for manual edits of the pipeline.sh to deploy for a custom idp

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- users can override the 3 parameters needed to deploy for a custom IDP without editing the pipeline.sh 
- minor simplification of the test config

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Comment by JoelGauci** [comment]
ok for this enhancement

File: references/identity-facade/README.md [hunk 1]
```diff
@@ -13,11 +13,10 @@ the IdP.
 
 The IdP that is used is the [OIDC Mock IdP](../oidc-mock) but you may
 use any OIDC compliant IdP. Should you use your own IdP solution, please
-modify the value of the variable `idp_discovery_document` in the
-`pipeline.sh` script. This variable MUST point to the URL of the
-discovery document of your IdP solution.
-Other values that need to be changed are values of the 2 following variables:
+privde the following environment variables to the pipeline.sh script:
```
Comment (JoelGauci) [inline references/identity-facade/README.md:16]: provide (prvide)

**Review by JoelGauci — APPROVED** [review]
I approve the changes

**Review by laughingbiscuit — APPROVED** [review]

**Pull request merged**